src/client/conn/http1.rs

```diff
@@ -97,6 +97,9 @@ where
 /// A builder to configure an HTTP connection.
 ///
 /// After setting options, the builder is used to create a handshake future.
+///
+/// **Note**: The default values of options are *not considered stable*. They
+/// are subject to change at any time
```

Comment:
Could you add a period to the end of the sentences?

Reply:
Done.